eclipse-scout-core/src/ajax/ajax.ts

```diff
@@ -26,7 +26,7 @@ export const ajax = {
   get(url: string, options?: JQuery.AjaxSettings, model?: AjaxCallModel): JQuery.Promise<any, AjaxError> {
     let opts = $.extend({}, {
       url: url,
-      type: 'GET'
+      method: 'GET'
```

Comment:
Consider a separate commit for this change, it has nothing to do with data objects.

Reply:
That's true. But the extra work to separate this now does not pay out here I think.